.travis.yml

```diff
@@ -1,11 +1,12 @@
 language: ruby
 
 rvm:
-  - 2.1.2
+  - 2.2.0
+  - 2.1.4
   - 2.0.0
   - 1.9.3
   - ruby-head
-  - jruby-19mode
+  - jruby-1.7.17
```

Comment:
Now you know why there are two commits! :-)

My first commit set `1.7.18`. I reasoning was the same as yours. Once that test run failed again I did some more Googling. That's when I found the 1.7.17 workaround. It's a very unintuitive issue.